### ISSUE TITLE
Accidentally corrupted db after stalker upgrade

### ISSUE THREAD

**Issue by fredrikaverpil** [eoyilmaz/stalker#32]
Hi,

I upgraded to the latest stalker version and performed a `db.setup(database_engine_settings)` which seems to have corrupted our database. It got stuck at "creating tables" and didn't move on from there:

```
DEBUG:stalker.db:settings: {'sqlalchemy.pool_size': 1, 'sqlalchemy.echo': False, 'sqlalchemy.url': u'postgresql+psycopg2://dbuser:***@1.2.3.4/stalkerdb', 'sqlalchemy.max_overflow': 3}
DEBUG:stalker.db:engine: Engine(postgresql+psycopg2://dbuser:***@1.2.3.4/stalkerdb)
DEBUG:stalker.db:creating the tables
```

I'm guessing I should have first performed an `alembic upgrade head` on the database?

Since time was of the essence (and it's Friday afternoon), I restored a database backup straight away, so all is back on track now. Phew! 😅 

However, I was wondering... Would it be possible to avoid this kind of thing in the future by perhaps throwing an error of some kind if the tables are not in the expected format, avoiding db corruption?

**Comment by eoyilmaz** [comment]
Yes I think we should do that, and it should be fairly easy by using the alembic_version table.

**Comment by fredrikaverpil** [comment]
Erkan, this is excellent - thank you!
